prover/zkevm/prover/hash/packing/packing_test.go

```diff
@@ -18,7 +18,7 @@ func makeTestCasePackingModule(uc generic.HashingUsecase) (
 	var (
 		// max number of blocks that can be extracted from limbs
 		// if the number of blocks passes the max, newPack() would panic.
-		maxNumBlock = 35
+		maxNumBlock = 36
```

Comment:
--- FAIL: TestPackingModule (0.13s)
    --- FAIL: TestPackingModule/MiMC (0.02s)
panic: the number of the blocks 36 passes the limit 35 [recovered]
	panic: the number of the blocks 36 passes the limit 35
	
@AlexandreBelling Do you have any idea what caused this change?

Reply:
I guess previously the `number of blocks` was counted as 35, but now it's counted as 36, which caused it to exceed the `maxNumBlock` of 35.

Reply:
Oh right I did a small change regarding this. I did not see it broke the tests

Reply:
I think it might have been skipped by the CI previously and this PRs makes it happen again